cinema/serializers.py

```diff
@@ -1,24 +1,27 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
 
 
-class MovieSerializer(serializers.Serializer):
-    id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+class MovieSerializer(serializers.ModelSerializer):
```

Comment:
I could be wrong, but it seems that the request was to implement the serializer following the more flexible serializers.Serializer rather than serializers.ModelSerializer. I understand that the functionality in this case is the same in them. However, it would be better to try to implement following serializer.Serializer.

Reply:
in this task it is not necessary to use Serializer class, since we are not customizing the functionality, but using the default, so it's good to use ModelSerializer here.